jlm/mlir/frontend/MlirToJlmConverter.hpp

```diff
@@ -150,6 +163,14 @@ class MlirToJlmConverter final
       rvsdg::Region & rvsdgRegion,
       const ::llvm::SmallVector<rvsdg::output *> & inputs);
 
+  /**
+   * Converts a floating point size to jlm::llvm::fpsize.
```

Comment:
Consistency, using FP in the other places, spelling it out like here is the better option

Reply:
Changed to spelling it out in both converter header files